### PR TITLE
feat(across): break down fees

### PR DESCRIPTION
**Pull request by Gamaranto**
**Motivation**
This PR breaks down the fees into bridge fees and ethereum gas fees

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: src/components/SendAction/SendAction.tsx [hunk 1]
```diff
@@ -151,17 +151,22 @@ const SendAction: React.FC = () => {
...
+              <div>Ethereum Gas Fee</div>
               <div>
                 {formatUnits(
-                  fees.instantRelayFee.total
-                    .add(fees.slowRelayFee.total)
-                    .add(fees.lpFee.total),
+                  fees.instantRelayFee.total.add(fees.slowRelayFee.total),
                   tokenInfo.decimals
                 )}{" "}
                 {tokenInfo.symbol}
```
Comment (Tulun) [inline src/components/SendAction/SendAction.tsx:160]: I might pull this out into a function call given how messy it's starting to look.

**Review by Tulun — APPROVED** [review]
Overall ok, just maybe refactor the view slightly.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Review by daywiss — APPROVED** [review]

File: src/components/SendAction/SendAction.tsx (hunk 1 above)
Comment (mrice32) [reply src/components/SendAction/SendAction.tsx:160]: Let's do this in a follow-up just so we can get this breakdown out.

**Pull request merged**